src/components/SplitVertical/SplitVertical.less

```diff
@@ -0,0 +1,47 @@
+@SplitVertical-transition-duration: @timing-animation-fade;
+@SplitVertical-transition-half-duration: @SplitVertical-transition-duration / 2;
+
+.lucid-SplitVertical {
+	overflow: hidden;
+
+	& > &-inner {
```

Comment:
Is there a reason you're doing so many direct descendent selectors? Is it to support nested panes?